tests/e2e/fixtures/helpers.ts

```diff
@@ -146,10 +146,9 @@ export async function loadContext(
   let context: BrowserContext | undefined;
   if (browserName === 'chromium') {
     context = await chromium.launchPersistentContext('', {
-      headless: true,
       channel,
       args: [
-        `--headless=true`,
+        `--headless=new`,
```

Comment:
It looks like when using `--headless=new`, the worker doesn't get stuck here:
https://github.com/interledger/web-monetization-extension/blob/49b9862372e71592b860dcc05228ec1280172728/tests/e2e/fixtures/helpers.ts#L207-L209

When using Chrome's `--headless=new` it will target version 109 and above and it allows us to get the full functionality of Chrome. With the initial setup we were targeting the versions between 96 and 109.

Old headless mode: https://developer.chrome.com/docs/chromium/headless#use_old_headless_mode